### PR TITLE
added tests for contacts and accounts

### PR DESCRIPTION
**Pull request by YurySuvalau**

### CODE REVIEW

File: src/test/java/elements/Search.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Search {
+
+    WebDriver driver;
+    String label;
+    String locator = "//*[contains(text(),'%s')]/ancestor::*[@class='slds-form-element__control']//input";
+    String selectOption = "//*[contains(text(),'%s')]/ancestor::div[contains(@role,'listbox')]";
```
Comment (vchesnakoutest) [inline src/test/java/elements/Search.java:11]: better to replace "div" to * (asterisk) if possible

File: src/test/java/objects/Account.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    String fax;
+    String industry;
+    String employees;
+    String annual_revenue;
+    String description;
+    String billing_street;
+    String billing_city;
+    String billing_state_province;
+    String billing_zip_postal_code;
+    String billing_country;
```
Comment (vchesnakoutest) [inline src/test/java/objects/Account.java:17]: variables should be in camelCase

File: src/test/java/page/AccountListPage.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        driver.findElement(NEW_BUTTON).click();
+    }
+
+    public boolean isAccountNameDisplayed() {
+        String label = driver.findElement(TITLE_NAME).getText();
+        if (label.contains(NAME_ACCOUNT)) {
+            return true;
+        }
+        return false;
+    }
```
Comment (vchesnakoutest) [inline src/test/java/page/AccountListPage.java:44]: should be in 'Chain Of Invocations' style

File: src/test/java/page/AccountListPage.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        openPage(URL_LOGIN + URL_ACCOUNT_LIST);
+    }
+
+    public void clickNewButton() {
+        driver.findElement(NEW_BUTTON).click();
+    }
+
+    public boolean isAccountNameDisplayed() {
+        String label = driver.findElement(TITLE_NAME).getText();
+        if (label.contains(NAME_ACCOUNT)) {
```
Comment (vchesnakoutest) [inline src/test/java/page/AccountListPage.java:40]: return label.contains(NAME_ACCOUNT)

File: src/test/java/page/ContactsListPage.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        try {
+            wait.until(ExpectedConditions.elementToBeClickable(NEW_CONTACT_BUTTON));
+        } catch (TimeoutException exception) {
+            Assert.fail(String.format("Home page it's not loaded! Locator: '%s' was not found!", NEW_CONTACT_BUTTON));
+        }
+        return this;
+    }
+
+    public void openPage() {
+        openPage(URL_LOGIN+URL_CONTACTS_PAGE);
```
Comment (vchesnakoutest) [inline src/test/java/page/ContactsListPage.java:33]: URL_LOGIN + URL_CONTACTS_PAGE

File: src/test/java/page/ContactsListPage.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+public class ContactsListPage extends BasePage {
+
+    public ContactsListPage(WebDriver driver) {
+        super(driver);
+    }
+
+    By NEW_CONTACT_BUTTON = By.cssSelector("[title=New]");
+    String NAME_CONTACT = contacts.getFirstName().split(" ")[0];
+    By TITLE_NAME = By.cssSelector("[title~="+NAME_CONTACT+"]");
```
Comment (vchesnakoutest) [inline src/test/java/page/ContactsListPage.java:20]: use proper code style: By.cssSelector("[title~=" + NAME_CONTACT + "]");

File: src/test/java/page/ContactsListPage.java [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    public void clickOnNewContactButton() {
+        driver.findElement(NEW_CONTACT_BUTTON).click();
+    }
+
+    public boolean isContactNameDisplayed() {
+        String label = driver.findElement(TITLE_NAME).getText();
+        if (label.contains(NAME_CONTACT)) {
+            return true;
+        }
+        return false;
```
Comment (vchesnakoutest) [inline src/test/java/page/ContactsListPage.java:45]: just return in one line

File: src/test/java/page/HomePage.java [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+    public void openPage() {
+        openPage(URL_LOGIN + URL_HOME_PAGE);
+    }
+
+    public void openAccountPage() {
+        wait.until(ExpectedConditions.elementToBeClickable(ACCOUNT_BUTTON));
+        JavascriptExecutor executor = (JavascriptExecutor) driver;
+        executor.executeScript("arguments[0].click();", driver.findElement(ACCOUNT_BUTTON));
+    }
```
Comment (vchesnakoutest) [inline src/test/java/page/HomePage.java:38]: better to open by URL

File: src/test/java/page/HomePage.java [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+    public void openContactsPage() {
+        wait.until(ExpectedConditions.elementToBeClickable(CONTACTS_BUTTON));
+        JavascriptExecutor executor = (JavascriptExecutor) driver;
+        executor.executeScript("arguments[0].click();", driver.findElement(CONTACTS_BUTTON));
+    }
```
Comment (vchesnakoutest) [inline src/test/java/page/HomePage.java:44]: better to open by URL

File: src/test/java/page/NewAccountModal.java [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+import org.testng.Assert;
+
+import java.util.ArrayList;
+
+public class NewAccountModal extends BasePage {
+    public NewAccountModal(WebDriver driver) {
+        super(driver);
+    }
+
+    By MODEL_PAGE_LABEL = By.xpath("//h2[contains(text(), 'New Account')]");
```
Comment (vchesnakoutest) [inline src/test/java/page/NewAccountModal.java:20]: MODAL - typo

File: src/test/java/page/NewContactsModal.java [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class NewContactsModal extends BasePage {
+    public NewContactsModal(WebDriver driver) {
+        super(driver);
+    }
+
+    By SELECT = By.cssSelector("[class~=select]");
+    By SAVE_BUTTON = By.xpath("//*[@class='button-container-inner slds-float_right']/button[3]");
```
Comment (vchesnakoutest) [inline src/test/java/page/NewContactsModal.java:20]: this locator should be written in such way:
//button[@title='Save']

**Review by vchesnakoutest — COMMENTED** [review]
please make fixes

File: src/test/java/elements/Search.java (hunk 1 above)
Comment (YurySuvalau) [reply src/test/java/elements/Search.java:11]: fix

File: src/test/java/objects/Account.java (hunk 2 above)
Comment (YurySuvalau) [reply src/test/java/objects/Account.java:17]: fix

File: src/test/java/page/AccountListPage.java (hunk 3 above)
Comment (YurySuvalau) [reply src/test/java/page/AccountListPage.java:44]: fix

File: src/test/java/page/ContactsListPage.java (hunk 5 above)
Comment (YurySuvalau) [reply src/test/java/page/ContactsListPage.java:33]: fix

File: src/test/java/page/NewAccountModal.java (hunk 10 above)
Comment (YurySuvalau) [reply src/test/java/page/NewAccountModal.java:20]: fix

File: src/test/java/page/HomePage.java (hunk 8 above)
Comment (YurySuvalau) [reply src/test/java/page/HomePage.java:38]: fix

File: src/test/java/page/HomePage.java (hunk 9 above)
Comment (YurySuvalau) [reply src/test/java/page/HomePage.java:44]: fix

File: src/test/java/page/NewContactsModal.java (hunk 11 above)
Comment (YurySuvalau) [reply src/test/java/page/NewContactsModal.java:20]: fix

**Pull request merged**